packages/@aws-cdk/aws-apigatewayv2/lib/http/route.ts

```diff
@@ -167,7 +165,7 @@ export class HttpRoute extends Resource implements IHttpRoute {
       routeKey: props.routeKey.key,
       target: `integrations/${integration.integrationId}`,
       authorizerId: authBindResult?.authorizerId,
-      authorizationType,
+      authorizationType: authBindResult?.authorizationType,
```

Comment:
To capture all cases and to safeguard against a future change where `authorizationType` from the bind result is made an optional prop, we should default to 'NONE' here.

```suggestion
      authorizationType: authBindResult?.authorizationType ?? HttpAuthorizerType.NONE, // must be explicitly NONE (not undefined) for stack updates to work correctly
```

Reply:
I'm wondering if there is any possible instance where someone would specifically want the `authorizationType` to be undefined as that is an option on the CloudFormation side.  If that actually is desired, the user could always `.addPropertyOverride('AuthorizationType', undefined)` to the resource node.